pkg/controller/infrastructure/configvalidator_test.go

```diff
@@ -188,6 +198,62 @@ var _ = Describe("ConfigValidator", func() {
 			}))
 		})
 
+		Describe("validate DHCP options", func() {
+
+			BeforeEach(func() {
+				awsClient.EXPECT().GetVPCAttribute(ctx, vpcID, "enableDnsSupport").Return(true, nil)
+				awsClient.EXPECT().GetVPCAttribute(ctx, vpcID, "enableDnsHostnames").Return(true, nil)
+				awsClient.EXPECT().GetVPCInternetGateway(ctx, vpcID).Return(vpcID, nil)
+			})
+
+			It("should allows VPC with correctly configurated DHCP options", func() {
```

Comment:
Can you add a tests that cover the case when the region is `us-east-1`?
Additionally, it would be better to use a DescribeTable() here to avoid code repetition.